pkg/api/handlers/compat/images.go

```diff
@@ -325,31 +326,25 @@ func CreateImageFromImage(w http.ResponseWriter, r *http.Request) {
 
 loop: // break out of for/select infinite loop
 	for {
-		var report struct {
```

Comment:
That was just a copy/paste of JSONMessage which is used in push endpoint anyway.